schemas/stsci.edu/asdf/transform/constant-1.4.0.yaml

```diff
@@ -0,0 +1,23 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/constant-1.4.0"
+tag: "tag:stsci.edu:asdf/transform/constant-1.4.0"
+title: >
+  A transform that takes no inputs and always outputs a constant
+  value.
```

Comment:
This is not quite right because it takes 1 or 2 inputs based on the dimensionality. Perhaps just say
"A Constant transform".
And add more to the description.